assignment.js

```diff
@@ -113,6 +149,55 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+// declaring variables
+let playerAChoice
+let playerBChoice
+
+// test case no. 1
+playerAChoice = "rock"
+playerBChoice = "paper"
+
+// rock, paper, scissors logic
+if (playerAChoice === "rock" && playerBChoice === "scissors" || playerAChoice === "scissors" && playerBChoice === "paper" || playerAChoice === "paper" && playerBChoice === "rock") {
```

Comment:
@nhtaylor nice work using a combination of logical And (`&&`) and logical Or (`||`) to solve this. Great job maintaining good indentation throughout.